decidim-regulations/app/cells/decidim/regulations/regulation_filters_cell.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Regulations
+    class RegulationFiltersCell < Decidim::ViewModel
```

Comment:
you could add the `self.translation_path = 'decidim.participatory_processes.participatory_processes.filters'` to DRY up `t()` in this file and all the views from `decidim-regulations/app/cells/decidim/regulations/regulation_filters/*.erb`

[Trailblazer: Cells and Rails # [Trailblazer: Cells and Rails](http://trailblazer.to/gems/cells/rails.html#translation-and-i18n-helper)](http://trailblazer.to/gems/cells/rails.html#translation-and-i18n-helper)

Reply:
I've tried without success.. Also in Decidim I haven't found anywhere where it is done something like this...